### PR TITLE
Form styles and editable form elements

### PR DESCRIPTION
**Pull request by fynnfeldpausch**

### CODE REVIEW

File: source/docs/index.html [hunk 1]
```diff
@@ -49,6 +49,99 @@ <h1 class="nav-brand">mindsmash UI</h1>
...
+          </div>
+        </form>
+      </div>
+
+      <div ng-controller="EditableTextController">
+        <button class="btn btn-default pull-right" ng-click="data.show = !data.show">Toggle</button>
+        <h2>Toggle State</h2>
+
+        <div class="mb-l">
+          <form msm-editable-form="data.show" class="form-horizontal">
```
Comment (badabam) [inline source/docs/index.html]: Improve wording: data.show

Comment (fynnfeldpausch) [inline source/docs/index.html]: fixed

File: source/components/ui/msm-editable-text/msm-editable-text.directive.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+(function () {
+  'use strict';
+
+  /**
+   * @ngdoc directive
+   * @name components.ui.msmEditableText
```
Comment (badabam) [inline source/components/ui/msm-editable-text/msm-editable-text.directive.js:6]: Isn't this also written `components.ui:msmEditableText`?

**Comment by badabam** [comment]
Merged.

**Pull request closed without merging**